opencsp/app/sofast/lib/DistanceOpticScreen.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass, field
+
+from opencsp.common.lib.geometry.Vxyz import Vxyz
+import opencsp.common.lib.tool.hdf5_tools as h5
+
+
+@dataclass
+class DistanceOpticScreen(h5.HDF5_IO_Abstract):
+    """Represents a distance measurement between the optic and the center of the screen. This measurement is typically
+    achieved by displaying the crosshairs from the SofastGUI and measuring from the center of the optic
+    (measurement_point 0,0,0) to the center of the crosshairs."""
+
+    v_measure_point_facet: Vxyz = field(default_factory=lambda: Vxyz(0.0, 0.0, 0.0))
```

Comment:
The xyz inputs into Vxyz are actually supposed to be a length 3 tuple. 
```suggestion
    v_measure_point_facet: Vxyz = field(default_factory=lambda: Vxyz((0.0, 0.0, 0.0)))
```

Reply:
Erg, whoops. Thanks! 🙏